### DISCUSSION TITLE
[VOTE] Release Apollo version 2.3.0

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#5195] [General]
Hi All,

@apolloconfig/pmc @apolloconfig/committers This is a call for vote to release Apollo version 2.3.0.

Release Note:

 * https://github.com/apolloconfig/apollo/releases/tag/v2.3.0

Release Candidate:

 * https://github.com/apolloconfig/apollo/releases/download/v2.3.0/apollo-configservice-2.3.0-github.zip
 * https://github.com/apolloconfig/apollo/releases/download/v2.3.0/apollo-adminservice-2.3.0-github.zip
 * https://github.com/apolloconfig/apollo/releases/download/v2.3.0/apollo-portal-2.3.0-github.zip
 * sha1 checksums
   - 572132fc874be095b5b47d79c1453ae1e761f442  apollo-portal-2.3.0-github.zip
   - b2ab0860614946fd8b026cf2acbf8f2cbc7b67eb  apollo-configservice-2.3.0-github.zip
   - 781d3510e6b4c27304664bf013dd63d7c6ee5da9  apollo-adminservice-2.3.0-github.zip

Release Tag :

 * https://github.com/apolloconfig/apollo/releases/tag/v2.3.0

Release CommitID :

 * https://github.com/apolloconfig/apollo/commit/c28eb49ac0f2bf5cb0f68fd3deccd192709f996f

Guide to build the release from source:

 * https://www.apolloconfig.com/#/en/deployment/distributed-deployment-guide?id=_2212-building-from-source-code

Voting will start now (2024/08/10) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by hezhangjian** [comment]
+1, I checked
- build from source successed
- unit tests passed on mac m1

**Comment by Anilople** [comment]
+1 Release this package.

I checked:
sha1 checksum
```bash
$ sha1sum apollo-*-2.3.0-*.zip
781d3510e6b4c27304664bf013dd63d7c6ee5da9 *apollo-adminservice-2.3.0-github.zip
b2ab0860614946fd8b026cf2acbf8f2cbc7b67eb *apollo-configservice-2.3.0-github.zip
572132fc874be095b5b47d79c1453ae1e761f442 *apollo-portal-2.3.0-github.zip
```

**Comment by zouyx** [comment]
+1 binding to release

I've checked 
- [x] All uts passed.
- [x] Release Note is fine.
- [x] Download links are fine.
- [x] Shasum are correct.

**Comment by klboke** [comment]
+1, I checked
 - All uts passed.
 - Release Note is fine.

**Comment by nobodyiam** [comment]
The vote to release Apollo version 2.3.0 has passed with four +1 and zero -1.
